Scripts/General/input_manager.gd

```diff
@@ -18,6 +18,16 @@ func init_game_setup():
 	game_setup_info = GameSetupInfo.create_empty()
 
 
+## called:
+## * by UI when button is clicked or game starts (host is true)
+## * by client when server orders, then host is false
+func init_battle_mode(host : bool):
+	game_setup_info.game_mode = GameSetupInfo.GameMode.BATTLE
+
+	if host:
```

Comment:
maybe here should be (not NET.client) instead of host, but let's fix it after fixing other bug